webapp/channels/src/packages/mattermost-redux/src/utils/channel_utils.ts

```diff
@@ -367,6 +367,7 @@ export function calculateUnreadCount(
     member: ChannelMembership | null | undefined,
     crtEnabled: boolean,
 ): {showUnread: boolean; mentions: number; messages: number; hasUrgent: boolean} {
+    // LOL
```

Comment:
LOL?

Reply:
Not sure what this is about - maybe he left that as a reminder to delete? I noticed the same here: https://github.com/mattermost/mattermost/pull/23952#discussion_r1255016576

Reply:
I'll just delete this pending files (I'm working on this PR as Harshil is out)

Reply:
This one is related to unreads so I don't think we need to delete the whole file. I'll just remove the comment.